db/schema.rb

```diff
@@ -120,6 +124,13 @@
     t.index ["code"], name: "unique_code", unique: true
   end
 
+  create_table "organizations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "title"
+    t.string "type"
```

Comment:
`type` is reserved for Single Table Inheritance. I often use `kind` instead, but whatever else sounds good to you (`organization_type` is a possibility too).